diff --git a/globals.d.ts b/globals.d.ts
index f9d5d3d7fb9606a5aaec2234d4139b1d61eaf84c..8a938814c9d579f92150b8cf1e97d552f7cf5c30 100644
--- a/globals.d.ts
+++ b/globals.d.ts
@@ -1,7 +1,7 @@
 /**
  * https://discord.com/developers/docs/reference#snowflakes
  */
-export type Snowflake = string;
+export type Snowflake = string & import("zod").BRAND<"Snowflake">;
 /**
  * https://discord.com/developers/docs/topics/permissions
  *
